packages/flame/lib/src/collisions/hitboxes/shape_hitbox.dart

```diff
@@ -10,7 +12,10 @@ import '../../geometry/shape_intersections.dart' as intersection_system;
 /// [PolygonHitbox].
 mixin ShapeHitbox on ShapeComponent implements Hitbox<ShapeHitbox> {
   @override
-  CollidableType collidableType = CollidableType.active;
+  CollisionType collisionType = CollisionType.active;
+
+  @override
```

Comment:
Is this intentional?

Reply:
To override the paint? Yes, because otherwise the hitboxes just became big blocks when they were rendered in `debugMode`. I guess it could override the debug render method too, I'll have a think about.